src/blocks/block-cta/index.js

```diff
@@ -240,7 +240,7 @@ class ABCTABlock extends Component {
 						tagName="span"
 						placeholder={ __( 'Button text...', 'atomic-blocks' ) }
 						value={ buttonText }
-						formattingControls={ [] }
+						allowedFormats={ [] }
```

Comment:
There was a console warning from `formattingControls` being changed to `allowedFormats`